cli/scripts/e2e/e2e.ts

```diff
@@ -48,7 +48,21 @@ for (let npmClient of ["yarn", "berry", "pnpm6", "pnpm", "npm"] as const) {
   const Suite = uvu.suite(`${npmClient}`);
 
   const repo = new Monorepo("basics");
-  repo.init(npmClient, basicPipeline);
+  repo.init(npmClient, {
+    pipeline: {
```

Comment:
There is maybe a better place to do this.

Reply:
This would be fine, but I'm going to ask for enough bonus tests that you're gonna want a `prunePipeline` constant. 😜